### DISCUSSION TITLE
srsue with open5gs

### DISCUSSION THREAD

**Discussion by zhibom** [srsran/srsRAN_4G#1146] [General]
Hi all,

I am writing to seek your assistance regarding an issue I am facing while installing srsRan 5G SA. I have successfully installed srsgnb and open5gs and connected them. However, when I try to connect srsue with srsgnb, I encounter the following problem:
It appears that srsue can connect with the phy layer, but it cannot be registered by open5gs. I have included the log information below:
   
![Screenshot from 2023-04-10 14-29-17](https://user-images.githubusercontent.com/130408333/231007744-7a40f2e0-12d6-4774-90b1-406dea892b0d.png)
![Screenshot from 2023-04-10 14-29-40](https://user-images.githubusercontent.com/130408333/231007747-fb012053-a2e2-451a-b7e0-2dbf428fe152.png)
![Screenshot from 2023-04-10 14-31-51](https://user-images.githubusercontent.com/130408333/231007748-8596bea3-094e-445e-8768-7aa16515ec88.png)

Here is the configuration of srsue and open5gs:
![Screenshot from 2023-04-10 14-57-58](https://user-images.githubusercontent.com/130408333/231007826-0c15da33-2b89-4c39-ab78-f8a0af9f479f.png)
![Screenshot from 2023-04-10 14-58-18](https://user-images.githubusercontent.com/130408333/231007827-d49d480a-2c74-4d38-a53b-2914e70787c2.png)
![Screenshot from 2023-04-10 14-59-59](https://user-images.githubusercontent.com/130408333/231007830-deda14c5-90f6-428f-a2e0-ea9a5d3e0698.png)

Thank you in advance for your time and assistance.

**Comment by andrepuschmann** [comment]
It looks like the UE can't complete the RRC Setup - there are many PRACHs. You have to investigate that first. Since you're using real radios double check gains etc.

**Comment by rajugbv** [comment]
Hi zhibom 
i am also facing related issue,so can please sent me the screenshot of the gnb configuration file
